evaluate.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/bash
 
+export DB_HOST=localhost
```

Comment:
Para corrigir todos os projetos fossem configurados para enviar `localhost` no lugar de `mongodb`, mas para não precisar alterar todos forço ser sempre localhost.